app/scripts/views/forms.coffee

```diff
@@ -971,6 +972,26 @@ define [
       @listenToNewFormView()
       @showNewFormViewAnimate()
 
+
+    # Request the history of the form model, the user can click on a revision and see the details
+    # preferably by simply showing the other revision as a model along side in the same views
+    fetchHistory: (formModel) ->
+
+      fielddbHelperModel = new (FieldDB.Datum)(_id: formModel.id)
+      fielddbHelperModel.fetchRevisions().then(((revisions) ->
```

Comment:
which version of fielddb is being used? 

```
new FieldDB.FieldDBObject().version
```

most likely your local version is old. and you don't have the #118 code so you didnt run the two new grunt functions i added to make your local version uptodate and symlinked